misc/bazel/cmake/cmake.bzl

```diff
@@ -49,7 +49,7 @@ def _get_includes(includes):
     return [_cmake_path(i) for i in includes.to_list() if "/_virtual_includes/" not in i]
 
 def _cmake_aspect_impl(target, ctx):
-    if not ctx.rule.kind.startswith("cc_"):
+    if not ctx.rule.kind.startswith(("cc_", "_cc_add_features")):
```

Comment:
Nit: Should we just have `_cc` as prefix?

Reply:
I thought about that, but then I did not want to catch other internal things from external libraries by chance... although maybe it would be bad or anything.